streams/src/test/java/org/apache/kafka/streams/integration/RegexSourceIntegrationTest.java

```diff
@@ -90,7 +90,7 @@
     private static final String PARTITIONED_TOPIC_1 = "partitioned-1";
     private static final String PARTITIONED_TOPIC_2 = "partitioned-2";
 
-    private static final String DEFAULT_OUTPUT_TOPIC = "outputTopic";
+    private static final String DEFAULT_OUTPUT_TOPIC_NAME = "outputTopic";
```

Comment:
We no longer need `DEFAULT_OUTPUT_TOPIC_NAME` if all caller will use `createTopic` now

Reply:
ack